eng/pipelines/templates/steps/analyze.yml

```diff
@@ -0,0 +1,81 @@
+parameters:
```

Comment:
I'm liking this file separation! A bit noisy for the PR, but a huge improvement for these files.